docs/Using Fleet/MDM-macOS-setup-experience.md

```diff
@@ -0,0 +1,345 @@
+
+* To deploy custom scripts, you need to generate a package. The [munkipkg tool](https://github.com/munki/munki-pkg) is a popular tool for generating packages.
+
+Apple requires that your package is a distribution package. Verify that the package is a distribution package:
+
+1. Run the following commands to expand you package and look at the files in the expanded folder:
+
+  ```bash
+  $ pkgutil --expand package.pkg expanded-package
+  $ ls expanded-package
+  ```
+
+  If your package is a distribution package should see a `Distribution` file.
+
+2. If you don't see a `Distribution` file, run the following command to convert your package into a distribution package.
+
+  ```bash
+  $ productbuild --package package.pkg distrbution-package.pkg
+  ```
+
+  Make sure your package is a `.pkg` file.
+
+### Step 2: sign the package
+
+To sign the package we need a valid Developer ID Installer certificate:
+
+1. Login to your [Apple Developer account](https://developer.apple.com/account).
+2. Follow Apple's instructions to create a Developer ID Installer certificate [here](https://developer.apple.com/help/account/create-certificates/create-developer-id-certificates).
+
+  > During step 3 in Apple's instructions, make sure you choose "Developer ID Installer." You'll need this kind of certificate to sign the package.
+
+  Confirm that certificate is installed on your Mac by opening the **Keychain Access** application. You should see your certificate in the **Certificates** tab.
+
+3. Run the following command in the **Terminal** application to sign your package with your Developer ID certificate:
+
+  ```bash
+  $ productsign --sign "Developer ID Installer: Your name (Serial number)" /path/to/package.pkg /path/to/signed-package.pkg
+  ```
+
+  You might be prompted to enter the password for your local account.
+
+  Confirm that your package is signed by running the following command:
+
+  ```bash
+  $ pkgutil --check-signature /path/to/signed-package.pkg
+  ```
+
+  In the output you should see that package has a "signed" status.
+
+### Step 3: upload the package to Fleet
+
+Fleet UI:
+
+1. Head to the **Controls > macOS settings > macOS setup > Bootstrap package** page.
+
+2. Choose which team you want to add the bootstrap package to by selecting the desired team in the teams dropdown in the upper left corner.
+
+3. Select **Upload** and choose your bootstrap package.
+
+fleetctl CLI:
+
+1. Upload the package to a storage location (ex. S3 or GitHub). During step 4, Fleet will retrieve the package from this storage location and host it for deployment.
+
+  > The URL must be accessible by the computer that uploads the package to Fleet.
+  > * This could be your local computer or the computer that runs your CI/CD workflow.
```

Comment:
The single bullet-point looks a little odd here:
```suggestion
  > This could be your local computer or the computer that runs your CI/CD workflow.
```